### PR TITLE
18 implement initial frontend

### PR DESCRIPTION
**Pull request by vangoh74**

### CODE REVIEW

File: frontend/src/components/Title.tsx [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import "./css/Title.css"
+
+export default function Title() {
+    return (
+        <div className={"titleContainer"}>
+            <h1> Capstone Texas Hold'em App </h1>
```
Comment (Boltzmann) [inline frontend/src/components/Title.tsx:7]: My 2 cents: Whether Capstone or not, its Texas Hold'em, isn't it? Also the app...why not game or better only "Texas Hold'em"?
```suggestion
            <h1>Texas Hold'em</h1>
```

**Review by Boltzmann — APPROVED** [review]
Nice code and a good idea!

Read the comments and feel free to change things.

File: frontend/src/components/TableCards.tsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+type TableCardsProps = {
+    card : Card
+}
+
+export default function TableCards({card} : TableCardsProps) {
+    return (
+        <>
+            <div>{card.rank} {card.suit}</div>
+        </>
```
[the remark this answers is not in the document]
Comment (danielschwarz-neuefische) [reply frontend/src/components/TableCards.tsx:14]: Gegebenenfalls noch die Einrückung anpassen.

**Review by danielschwarz-neuefische — APPROVED** [review]
Habe es lokal getestet und duchgelesen. 👍

**Pull request merged**